frontend/src/tests/application-tests.js

```diff
@@ -1,3 +1,74 @@
 // TODO: Remove eslint disable
 // eslint-disable-next-line
-export function applicationsTests({ apiGET, apiPOST }) {}
+import PropTypes from "prop-types";
+import {
+    applicationPropTypes,
+    expect,
+    beforeAll,
+    it,
+    checkPropTypes
+} from "./utils";
+import { databaseSeeder } from "./setup";
+/**
+ * Tests for the API. These are encapsulated in a function so that
+ * different `apiGET` and `apiPOST` functions can be passed in. For example,
+ * they may be functions that make actual requests via http or they may
+ * be from the mock API.
+ *
+ * @param {object} api
+ * @param {Function} api.apiGET A function that when passed a route will return the get response
+ * @param {Function} api.apiPOST A function that when passed a route and data, will return the post response
+ */
+export function applicationsTests(api) {
+    const { apiGET, apiPOST } = api;
+    let session = null,
+        applicant = null;
+
+    const newApplicationData = {
+        comments: "",
+        program: "Phd",
+        department: "Computer Science",
+        previous_uoft_ta_experience: "Last year I TAed a bunch",
+        yip: 2,
+        annotation: "",
+        position_preferences: [
+            {
+                preference_level: 2,
+                position_id: 10
```

Comment:
is it better if we can use the real position data for this?

Reply:
We absolutely should not be hardcoding this...you have access to the seeded data through databaseSeeder.